pages/components/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Image from "next/image";
+import logo from '../../../assets/logo.png';
+
+
+const Footer = () => {
+    return (
+        <footer style={{ width: "100%", position: "relative", bottom: "0"}}>
```

Comment:
Hi @YuriDevAT. I was struggling with moving it to the bottom `position: relative;` So I found [this tutorial](https://dev.to/gedalyakrycer/ohsnap-this-how-to-pin-footers-in-react-3k1k) to help me.

Reply:
Thanks for sharing. The tutorial is good! Is it working as expected on your side? As mentioned in the tutorial, the position attribute for the footer depends on the settings of the main component. Since you did not make any changes there I guess it will not work.